src/Build/BackEnd/Shared/BuildResult.cs

```diff
@@ -519,10 +508,8 @@ public void MergeResults(BuildResult results)
                 // cached results after the first time the target is built.  As such, we can allow "duplicates" to be merged in because there is
                 // no change.  If, however, this turns out not to be the case, we need to re-evaluate this merging and possibly re-enable the
                 // assertion below.
-#if false
-                // Allow no duplicates.
-                ErrorUtilities.VerifyThrow(!HasResultsForTarget(targetResult.Key), "Results already exist");
-#endif
+                // ErrorUtilities.VerifyThrow(!HasResultsForTarget(targetResult.Key), "Results already exist");
```

Comment:
>// [](start = 16, length = 2)

remove whole line

Reply:
The only reason I didn't is due to the code comment above the line.

Reply:
Got it, go ahead and resolve

---
In reply to: [230223987](https://github.com/Microsoft/msbuild/pull/3895#discussion_r230223987) [](ancestors = 230223987)